packages/types-eslint-plugin-react-hooks/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "@types/eslint-plugin-react-hooks",
+    "version": "0.1.0",
+    "description": "TypeScript definitions for eslint-plugin-react-hooks",
+    "homepage": "https://github.com/cloudflightio/cloudflight-eslint-plugin",
+    "bugs": {
+        "url": "https://github.com/cloudflightio/cloudflight-eslint-plugin/issues"
+    },
+    "private": true,
+    "author": "Cloudflight GmbH",
+    "license": "Apache-2.0",
+    "main": "",
+    "types": "index.d.ts",
+    "dependencies": {
+        "@types/eslint": "*"
+    },
+    "typeScriptVersion": "4.5"
```

Comment:
Looks like this already existing in previous PRs. I am have seen the TS version specified like this and it's also not in sync with the TS Version we are actually using

Reply:
Fixed.